### PR TITLE
Update readme to include Dev Volume Docker instructions

### PR DESCRIPTION
**Pull request by nikhitasharma**
### Summary
Update readme to include Dev Volume Docker instructions

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +115,23 @@ npm run test
         "wdio-lowlight": "terra wdio --disableSeleniumService=true --themes clinical-lowlight-theme"
         }
        ```
+7. To terminate a dev container:
+    - Use command line or Rancher/Docker Desktop OR
+    - Using Visual Studio Code
+      -  Select the Remote Explorer icon in the Activity Bar or View -> Command Palette and run **Remote Explorer: Focus on Containers View**
+      - Locate the **terra-core_devcontainer** or currently running dev container under "Dev Containers"
```
Comment (trandrew1023) [inline README.md:122]: *nitpick, terra-core can be updated here

File: README.md [hunk 2]
```diff
@@ -109,6 +115,23 @@ npm run test
...
+        - You can also select **Rebuild Container** to restart the dev container
+8. To reopen a dev container:
+    - Existing local workspace (for Option 1)
+      - Open the project in Visual Studio Code
+      - Ensure the workspace contains the .devcontainer folder
+      - Navigate to View -> Command Palette and run **Dev Containers: Open Workspace in Container**
+    - Isolated dev container volume (for Option 2)
+      - Open Visual Studio Code
+      - Use the Remote Explorer icon in the Activity Bar or View -> Command Palette and run **Remote Explorer: Focus on Containers View** to view containers
+      - Locate the **terra-core_devcontainer** under "Dev Containers"
```
Comment (trandrew1023) [inline README.md:133]: *nitpick, terra-core can be updated here

File: README.md (hunk 1 above)
Comment (nikhitasharma) [reply README.md:122]: Yes, I missed that somehow!

File: README.md [hunk 3]
```diff
@@ -109,6 +115,23 @@ npm run test
         "wdio-lowlight": "terra wdio --disableSeleniumService=true --themes clinical-lowlight-theme"
         }
        ```
+7. To terminate a dev container:
+    - Use command line or Rancher/Docker Desktop OR
+    - Using Visual Studio Code
+      -  Select the Remote Explorer icon in the Activity Bar or View -> Command Palette and run **Remote Explorer: Focus on Containers View**
```
Comment (lqdev00) [inline README.md:121]:
```suggestion
      - Select the Remote Explorer icon in the Activity Bar or View -> Command Palette and run **Remote Explorer: Focus on Containers View**
```

**Review by lqdev00 — APPROVED** [review]

**Pull request merged**